### ISSUE TITLE
[Bug] Supabase Integration - "No API key found in request"

### ISSUE THREAD

**Issue by David-Kasilia** [AppFlowy-IO/AppFlowy#4313]
### Bug Description

I am facing an issue with the Superbase integration in Windows 10 Pro AppFlowy When attempting to interact with Superbase, I receive the following error message:
```
{
"message": "No API key found in request",
"hint": "No `apikey` request header or url param was found."
}
```
Though The Providers are working Correctly and its authenticated in the Superbase Authentication

### How to Reproduce

Is There a Base URL that is to be used after successful Authentication?

### Expected Behavior

The URL should redirect back to the application after Superbase authentication.

### Operating System

Windows 10 Pro

### AppFlowy Version(s)

v0.4.0

**Comment by appflowy** [comment]
![image](https://github.com/AppFlowy-IO/AppFlowy/assets/86001920/592aa3e3-a2a3-4219-a12f-0a560d2e82b9)

@David-Kasilia Did you enter the server information?

**Comment by David-Kasilia** [comment]
@appflowy , Yes I did Add The credentials correctly, The AUTH worked correctly but It was not redirecting back to App Flowy after successful Authentication

**Comment by LucasXu0** [comment]
> @appflowy , Yes I did Add The credentials correctly, The AUTH worked correctly but It was not redirecting back to App Flowy after successful Authentication

Did you set up the callback URL(appflowy-flutter://login-callback) in the Supabase website?

<img width="1018" alt="Screenshot 2024-01-08 at 11 21 40" src="https://github.com/AppFlowy-IO/AppFlowy/assets/11863087/bb1638d0-31eb-4ed6-9f28-1b06a22ddd01">

**Comment by David-Kasilia** [comment]
Thank you, @LucasXu0 for your response.  Initially, I had not added the redirect URL. I have just added the URL

I am facing a new error below is the error

![AppFlowy 1_8_2024 6_52_45 PM (1)](https://github.com/AppFlowy-IO/AppFlowy/assets/83514256/f6a3556a-d618-42be-a28f-1fbf33bed865)

**Comment by LucasXu0** [comment]
> Thank you, @LucasXu0 for your response.  Initially, I had not added the redirect URL. I have just added the URL
> 
> 
> 
> I am facing a new error below is the error
> 
> 
> 
> ![AppFlowy 1_8_2024 6_52_45 PM (1)](https://github.com/AppFlowy-IO/AppFlowy/assets/83514256/f6a3556a-d618-42be-a28f-1fbf33bed865)
> 
> 
> 
> 

cc @appflowy
